src/pages/Login/Login.css

```diff
@@ -1 +1,56 @@
+.mainContainer{
+    height: 100vh;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.cardContainer {
+    width: 45%;
+    height: 350px;
+}
+
+.cardContent {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+
+.cardText {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.heading {
+    margin-top: 0;
+    margin-bottom: 0px;
+    color: var(--orange);
+    font-size: 500%;
+    font-weight: 500;
+}
+
+.buttonContainer {
+    width: 55%;
+    height: 60px;
+    margin-top: 15px;
+    display: flex;
+    flex-direction: row;
+    gap: 20px;
+}
+
+.buttonText {

```

Comment:
for button text, you could use a unit called "em" to change based on the size of its parent container. For example,
.buttonText {
    font-size: 1 em;
}
This makes it so that when your webpage is smaller, your button gets smaller, and your text also gets smaller.

Reply:
Also make the font smaller. It looks too big compared to the size of the button